frontend/src/home.js

```diff
@@ -1,112 +1,143 @@
 import React from "react";
 import ReactDOM from "react-dom";
 
-function createMap() {
-  fetch("/charging-station-data")
-    .then(function(response) {
-      return response.json();
-    })
-    .then(chargingStations => {
-      const map = new google.maps.Map(document.getElementById("map"), {
-        center: { lat: 37.422, lng: -122.084 },
-        zoom: 7
-      });
-
-      chargingStations.forEach(chargingStation => {
-        addLandmark(
-          map,
-          chargingStation.lat,
-          chargingStation.lng,
-          "EV Charging Station",
-          "Electric Charging Station Located Here!"
-        );
-      });
-    });
-}
-
-/** Adds a marker that shows an info window when clicked. */
-function addLandmark(map, lat, lng, title, description) {
-  const marker = new google.maps.Marker({
-    position: { lat: lat, lng: lng },
-    map: map,
-    title: title
-  });
-  const infoWindow = new google.maps.InfoWindow({
-    content: description
-  });
-  marker.addListener("click", function() {
-    infoWindow.open(map, marker);
-  });
-}
-
-function createData(
-  id,
-  owner,
-  title,
-  description,
-  latitude,
-  longitude,
-  timestamp
-) {
-  return { id, owner, title, description, latitude, longitude, timestamp };
-}
-
 const rows = [
   createData(
```

Comment:
i don't think there's anything wrong with this and i'm just not very knowledgeable on this, but i was wondering what made you want to do test data instead of directly starting to posting places to the backend via frontend??

Reply:
I think we decided to do test points because I didnt think the datastore was ready for pins yet? but I might be wrong. I think thats the next step for sure though.

Reply:
yea that's true, sorry about that. i've been waiting for a few days and replied to all of tylers feedback but he hasn't gotten back to me. i nudged him, but i guess he must be busy

Reply:
Let's keep the test points